test/test_sparqlupdatestore.py

```diff
@@ -6,7 +6,7 @@
 from rdflib import ConjunctiveGraph, URIRef, Literal, BNode, Graph
 from urllib.request import urlopen
 
-HOST = "http://localhost:3031"
+HOST = "http://localhost:3030"
```

Comment:
I don't think this should change, or if it should I'm not sure why.

Reply:
Oversight on my part, will revert. (Local test detritus, result of struggles with fuseki, now resolved with your help, thanks.)